Makefile

```diff
@@ -2,15 +2,17 @@ CURPATH=$(PWD)
 TARGET_DIR=$(CURPATH)/_output
 
 GOBUILD=go build
-GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
+#GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
```

Comment:
This should be BUILD_GOPATH - it is only used when building the image - see below

Reply:
then you can change https://github.com/openshift/elasticsearch-operator/blob/be6866a901753ccea43a9a9cac259f7466307ad2/Makefile#L63
`  @GOPATH=$(BUILD_GOPATH) ....`

Reply:
Updated, thanks.